packages/syslog/syslog.2.0.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+
+synopsis: "syslog(3) routines for ocaml"
+
+authors: [
+  "Shawn Wagner"
+  "Eric Stokes"
+]
+
+maintainer: [ "Julien Sagot <julien.sagot@geneanet.org>" ]
+
+homepage: "http://github.com/geneanet/ocaml-syslog"
+dev-repo: "git://github.com/geneanet/ocaml-syslog"
+bug-reports: "https://github.com/geneanet/ocaml-syslog/issues"
+
+license: "LGPL-2.0-or-later"
+
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+
+depends: [
+  "dune" { build }
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
Dune should not be tagged as a build dependency (see #12466). Moreover according to your dune-project file, this package requires at least dune 2.0